data/store/scripts/outfits.lua

```diff
@@ -0,0 +1,79 @@
+-- Load outfits
+local outfits = {}
+
+for line in io.lines('data/XML/outfits.xml') do
```

Comment:
:sweat:
Help me with this, I'm sharing my time at work to do some contributions, is very little time :(